common/api-review/firestore-lite.api.md

```diff
@@ -18,12 +18,12 @@ export type AddPrefixToKeys<Prefix extends string, T extends Record<string, unkn
 };
 
 // @public
-export class AggregateField<T> {
-    type: string;
+export class AggregateField<R> {
+    readonly type = "AggregateField";
```

Comment:
Making this readonly is technically a breaking change, but no one should ever have changed this property.